docs/customization/model.rst

```diff
@@ -94,7 +96,21 @@ The file should be placed in ``AppBundle/Resources/config/doctrine/Country.orm.y
                 classes:
                     model: AppBundle\Entity\Country
 
-4. Additionally if you want to give the administrator an ability to add a ``flag`` to any of Countries,
+4. Update tha database:
+
+* via schema:
+
+.. code-block:: bash
+
+    $ php app/console doctrine:schema:update --force
+
+* via migrations:
+
+.. code-block:: bash
+
+    $ php app/console doctrine:migrations:migrat
```

Comment:
`doctrine:migrations:migrate` !== `doctrine:schema:update --force`

We should emphasize that we recommend a migration way, and present whole process:
```bash
$ doctrine:migrations:diff
$ doctrine:migrations:migrate
```